web/src/components/Nav.tsx

```diff
@@ -78,6 +79,7 @@ export const Nav = () => {
   const isSimulatorEnabled = useRecoilValue(simulatorEnabled);
   const [isSmallScreen] = useMediaQuery("(max-width: 640px)");
   const { session } = useSession();
+  useAnalytics();
```

Comment:
why is this in nav? to me it makes more sense in <App>

Reply:
Sorry, I forgot to add a comment about this.

I had it in App but I have this error ("App suspended while rendering, but no fallback UI was specified."):
![image](https://user-images.githubusercontent.com/72758386/197757053-fd4e951b-3f3a-4c84-ba74-1db727ac9787.png)

It happens because it is outside the Suspense component. I couldn't find a very straightforward solution for this. Should I wrap it in a component? Should I run it only when the component is ready (not sure how)? I could use some help

Reply:
I'm still having the error...